File: Easy_C++/Sum_of_Primes/Sum_of_Primes.pl
#Sample code to read in test cases:
use strict;
use warnings;

sub isprime
{
    my $num = shift;
    if ($num < 2) {return 0;}
    if ($num == 2) {return 1;}
    else 
    {   
        for (my $j = 2; $j <= sqrt($num); $j++) 
        {
            if ($num % $j == 0) {return 0;}
        }
    }
    return 1;
}

my $count = 0;
my $sum = 0;
my $n = 0;

while ($count < 1000)
{
    if(isprime($n))
    {
        $count++;
        $sum = $sum + $n;
    }
    $n++;
}
print $sum."\n";

File: Easy_C++/Sum_of_Primes/Sum_of_Primes.py
import sys
def isprime(n):
    # make sure it is a positive integer
    n = abs(int(n))
    if n < 2:
        return False
    if n == 2:
        return True
    if not n & 1:
        return False
    for x in range(3, int(n**0.5) + 1, 2):
        if n % x == 0:
            return False
    return True
    
count = 0
sum = 0
n = 0
while count < 1000:
    if isprime(n):
        count = count + 1
        sum = sum + n
    n = n + 1
print(sum)
    